docs/reference/docs/index_.asciidoc

```diff
@@ -293,22 +293,22 @@ replication succeeded/failed.
 === Refresh
 
 Control when the changes made by this request are visible to search. See
-<<docs-refresh,refresh>>.
+<<docs-refresh,`?refresh`>>.
```

Comment:
I think we should stick with the existing reference here. To me especially the leading question mark is confusing.